bears/r/FormatRBear.py

```diff
@@ -1,24 +1,22 @@
-from coalib.bearlib.abstractions.Lint import Lint
-from coalib.bears.LocalBear import LocalBear
+from coalib.bearlib.abstractions.Linter import linter
+from coalib.parsing.StringProcessing.Core import escape
 
 
-class FormatRBear(Lint, LocalBear):
-    executable = "Rscript"
-    arguments = ("-e 'library(formatR)' "
-                 "-e 'formatR::tidy_source(\"{filename}\")'")
-    prerequisite_command = ['Rscript', '-e', "'library(formatR)'"]
-    prerequisite_fail_msg = "Please install formatR for this bear to work."
-    diff_message = "Formatting can be improved."
-    gives_corrected = True
+@linter(executable='Rscript',
+        output_format='corrected',
+        prerequisite_check_command=('Rscript', '-e', 'library(formatR)'),
+        prerequisite_check_fail_message='Please install formatR for this bear '
+                                        'to work.')
+class FormatRBear:
+    """
+    This bear checks and corrects formatting of R Code using known formatR
+    utility.
+    """
```

Comment:
I think we normally do a newline here, though this is probably a matter of taste. Keep or change as you like.